src/sysen1/netime.128

```diff
@@ -150,7 +185,7 @@ wait:	20.*60.			; Wait 20 seconds polling other hosts.
 numerq:	2			; Ceiling of 2/3 of the hosts that respond
 denomq:	3			; must agree on the time.
 
-minq:	4			; Insist on at least 4 hosts with times in
+minq:	3			; Insist on at least 4 hosts with times in
```

Comment:
Make the comment match the value. But really, shouldn't this be a function of the length of the list of servers?

Reply:
If the number of servers is less than minq, then minq is adjusted to be that number.  So I think it is fine.  However, I wasn't reliably able to get times when I used minq with the list of hosts. So I think 3 is good. I'm fixing the comment.